tests/functional/nfs_feature/test_nfs_feature_enable_for_ODF_clusters.py

```diff
@@ -61,17 +64,31 @@ def test_nfs_not_enabled_by_default(self):
 
         Steps:
         1:- Check cephnfs resources not available by default
-
+        2:- Validate no NFS pods exist
         """
+
         storage_cluster_obj = ocp.OCP(
             kind="Storagecluster", namespace=config.ENV_DATA["cluster_namespace"]
         )
         # Checks cephnfs resources not available by default
+        log.info("Checking if any cephnfs resources exist")
         cephnfs_resource = storage_cluster_obj.exec_oc_cmd("get cephnfs")
         if cephnfs_resource is None:
-            log.info("No resources found in openshift-storage namespace.")
+            log.info("No cephnfs resources found. NFS should be disabled.")
+            pod_objs = get_all_pods(namespace=constants.OPENSHIFT_STORAGE_NAMESPACE)
+
+            nfs_pod_patterns = ("openshift-storage.nfs",)
+
```

Comment:
Instead of using a locally initiated value,
provisioner_selectors() method from ocs_ci/utility/nfs_utils.py can be used to get nfs-provisioner pod selectors.